tests/unit/components/sl-calendar-test.js

```diff
@@ -38,7 +38,7 @@ test( 'Locked property applies class', function( assert ) {
 });
 
 test( 'Lock mode prevents changing state', function( assert ) {
-    var component = this.subject({ locked: true }),
+    let component = this.subject({ locked: true }),
```

Comment:
There should be one assignment per line, per https://github.com/softlayer/ember-style-guide/blob/master/javascript.md#variables